spritesh.gemspec

```diff
@@ -0,0 +1,12 @@
+Gem::Specification.new do |s|
+  s.name        = "spritesh"
+  s.version     = "1.0.7"
```

Comment:
I accidentally pushed version 1.0.6 and I'm not sure how to overwrite it to be honest. RubyGems doesn't like you mucking with the versions. So I had to push v1.0.7 /: